numpy/lib/index_tricks.py

```diff
@@ -282,7 +284,7 @@ class OGridClass(nd_grid):
 
     See Also
     --------
-    np.lib.index_tricks.nd_grid : class of `ogrid` and `mgrid` objects
+    lib.index_tricks.nd_grid : class of `ogrid` and `mgrid` objects
```

Comment:
OK, to keep it here.  Maybe it is semi-public, although I am not actually sure there is a good reason for that.  I don't think direct use adds any real functionality.

Reply:
Yeah, I think we might as well remove it, there is no reason for direct access.